.github/workflows/runner.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
   windows-build:
     if: ${{ vars.DISABLE_GODOT_CI == '' }}
-    name: 🏁 Windows
+    name: 🪟 Windows
```

Comment:
I prefer the flag personally.
```suggestion
    name: 🏁 Windows
```

Reply:
I didn't know about that! Thanks, Aaron!